src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -4793,6 +4793,24 @@
      * Its children are the name of the member, the assignment literal
      * and the (compile-time constant conditional expression) value.
      *
+     * <p>For example:</p>
+     * <pre>
+     * &#064;MyAnnotation(someField1 = "Hello")
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * ANNOTATION
+     *  |--AT (&#064;)
```

Comment:
you missing ` -&gt; `

please look at example for ANNOTATION_ARRAY_INIT

Reply:
@romani , I changed the example this time and added the AST for that, previous example was a little bit confusing 😅, please check this.